package.json

```diff
@@ -16,10 +16,11 @@
         "pretty": "prettier --check scripts/**/*.js",
         "prettify": "prettier --write scripts/**/*.js",
         "lint-md": "remark -q -f content",
+        "lint": "eslint .",
```

Comment:
I wonder, should it be called something else? Like `lint-js`. 
After all, once Stumptown becomes a production success, the majority of this project will be about the markdown files and stuff. The scripting is just a small part.